src/shared/utilities.js

```diff
@@ -63,20 +63,42 @@ export const makePostHeader = (body) => {
   }
 
   return {
-    'method': 'POST',
-    'headers': {
-      'Accept': 'application/json',
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
       'Content-Type': 'application/json',

```

Comment:
See, this is exactly why I advocate quotes on ALL keys, because now it's inconsistent from this line

I won't say change it back but it seems unnecessary to have changed it here anyway :v